docs/_posts/2020-09-03-leaflet-1.7.1.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Announcing Leaflet 1.7.1
```

Comment:
The "Announcing" in every title gets a little weird... Maybe let's just use `Leaflet 1.7 released` as the title? Also maybe worth adding a note about us having a hiccup with `1.7.0` and hence `1.7.1`.

Reply:
I have read all the Leaflet code。and it is well designed.then i have a question：can we add webgl to the leaflet,and 3D？